packages/plugin/src/theia.d.ts

```diff
@@ -9701,5 +9701,140 @@ declare module '@theia/plugin' {
 		 */
         provideCallHierarchyOutgoingCalls(item: CallHierarchyItem, token: CancellationToken): ProviderResult<CallHierarchyOutgoingCall[]>;
     }
+
```

Comment:
Not sure why the API is separated into main and proposed.

Reply:
It comes from the vscode https://github.com/microsoft/vscode/commit/12f63d9c66b8a4bc9efca936d174184893f652a2

Reply:
Yes, but in VS Code it's all in vscode.d.ts, not vscode-proposed.d.ts anymore.

Reply:
I think they moved only consumer part to stable API

Reply:
@tsmaeder There is still a part of the authentication API in the `vscode.proposed.d.ts`: https://github.com/microsoft/vscode/blob/1.48.2/src/vs/vscode.proposed.d.ts#L21-L145